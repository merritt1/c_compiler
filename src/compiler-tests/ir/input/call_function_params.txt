void functionName(int a, int b) {}
void main(void) {
	int i, j;
	functionName(i, j);
}