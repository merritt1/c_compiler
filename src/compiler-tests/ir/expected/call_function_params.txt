(procBegin,  _Global_functionName)
(procEnd ,  _Global_functionName)
(procBegin,  main)
(addressOf,     r0,      i)
(loadWord,     r1,      r0)
(parameter,         0,      r1)
(addressOf,     r2,      j)
(loadWord,     r3,      r2)
(parameter,         1,      r3)
(call    ,  _Global_functionName)
(procEnd ,  main)
